scientific_library/tvb/datatypes/sensors.py

```diff
@@ -38,13 +38,18 @@
 """
 
 import re
+from enum import Enum
+
 import numpy
 from tvb.basic.readers import FileReader, try_get_absolute_path
 from tvb.basic.neotraits.api import HasTraits, Attr, NArray, Int
 
-EEG_POLYMORPHIC_IDENTITY = "EEG"
-MEG_POLYMORPHIC_IDENTITY = "MEG"
-INTERNAL_POLYMORPHIC_IDENTITY = "Internal"
+
+class SensorTypes(Enum):
+    TYPE_EEG = "EEG"
+    TYPE_MEG = "MEG"
+    TYPE_INTERNAL = "Internal"
+    TYPE_SEEG = "SEEG"
```

Comment:
`SEEG` and `Internal` are synonyms. We should only keep one of these. I propose `TYPE_INTERNAL`